docs/discounts/configure_discounts.md

```diff
@@ -30,6 +30,35 @@ ibexa:
                     products_list_per_page_limit: 15
 ```
 
+## Discount re-indexing
+
+Discounts feature uses [[= product_name_base =]] Messenger to reindex discounts and product prices in the background.
+This way changes are processed efficiently without slowing down the system and disrupting the user experience.
+
+When triggered periodically, the `ibexa:discounts:reindex` command identifies discounts that require re-indexing, ensuring prices always remain up-to-date.
+If there are edits to discounts that should result in changed product catalog prices, messages are dispatched to the [[= product_name_base =]] Messenger's queue and consumed by a background worker.
+The worker passes the messages to the handler, which then starts the re-indexing process at the most convenient moment.
+
+To run discount re-indexing in the background:
+
+1\. Make sure that the transport layer is [defined properly](background_tasks.md#configure-package) in [[= product_name_base =]] Messenger configuration.
+
+2\. Make sure that the worker starts together with the application to watch the transport bus:
```

Comment:
We may have to suggest that this Messenger command is not related to Discounts but to all background tasks. Maybe just with a link:
```suggestion
2\. Make sure that the [worker starts](background_tasks.md#start-worker) together with the application to watch the transport bus:
```